package.json

```diff
@@ -6,19 +6,21 @@
   "homepage": "https://github.com/CityOfZion/neon-wallet",
   "author": "Ethan Fast <ejhfast@gmail.com> (https://github.com/Ejhfast)",
   "scripts": {
-    "postinstall": "electron-builder install-app-deps",
+    "postinstall": "electron-builder install-app-deps && npm run electron-rebuild",
     "start": "electron .",
+    "start-dev": "electron . --enable-logging --remote-debugging-port=9222",
     "assets": "webpack --config ./config/webpack.config.prod",
     "assets-dev": "webpack --config ./config/webpack.config.dev",
     "assets-watch": "node_modules/.bin/webpack-dev-server --config ./config/webpack.config.dev  --port 3000 --host 0.0.0.0 --inline",
-    "test": "jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "test": "npm rebuild node-hid && jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```

Comment:
yarn add node-hid && jest...

Reply:
switching from npm to yarn causes the tests to fail, as yarn does not rebuild the native libraries.

Module version mismatch. Expected 48, got 57.

yarn "Says" it is rebuilding the binaries, but it does not.

`yarn test`

`
Test Suites: 7 failed, 7 passed, 14 total
`
`
 FAIL  __tests__/store/reducers.test.js
  ● Test suite failed to run

    Module version mismatch. Expected 48, got 57.
        at Error (native)
```

Reply:
I pulled your branch onto my local and it is working for me. I will ping you on slack. Think I know how to fix your problem.